src/components/button/button-types.style.js

```diff
@@ -100,6 +100,30 @@ export default ({ colors, disabled }, isDisabled, destructive) => ({
       }
     ` : ''}
   `,
+  dashed: `
+  background: transparent;
+  border: 2px solid ${colors.dashedBorder}
+  border-style: dashed;
+  color: ${colors.dashedButtonText};
+  &:hover {
+    background-color: ${colors.dashedHoverBackground}
+  }
```

Comment:
could you indent that block to make it more readable?

Reply:
Done.